Source/WebKit/WebProcess/GPU/graphics/RemoteDisplayListRecorderProxy.cpp

```diff
@@ -75,11 +75,15 @@ ALWAYS_INLINE void RemoteDisplayListRecorderProxy::send(T&& message)
 
     m_imageBuffer->backingStoreWillChange();
     auto result = m_renderingBackend->streamConnection().send(WTFMove(message), m_destinationBufferIdentifier, RemoteRenderingBackendProxy::defaultTimeout);
+#if !RELEASE_LOG_DISABLED
     if (UNLIKELY(result != IPC::Error::NoError)) {
         auto& parameters = m_renderingBackend->parameters();
         RELEASE_LOG(RemoteLayerBuffers, "[pageProxyID=%" PRIu64 ", webPageID=%" PRIu64 ", renderingBackend=%" PRIu64 "] RemoteDisplayListRecorderProxy::send - failed, name:%" PUBLIC_LOG_STRING ", error:%" PUBLIC_LOG_STRING,
```

Comment:
@smfr @kkinnunen-apple Shouldn't all this release logging in this file use RELEASE_LOG_ERROR?